usagov/vote-gov
# Vote-483: Update Arabic In English to be within link.

Ticket: [Vote-483](https://cm-jira.usa.gov/browse/VOTE-483)

Purpose:  This PR will move the `(In English)` string within the link to show that the external link will take the user to an English page. 

To Test:

1. visit [Preview Page](https://cg-9e8debaf-b030-4825-a43c-cb2bc850c96c.sites.pages.cloud.gov/preview/usagov/vote-gov/bug/vote-483-ar-link/ar/accessibility/#our-standards) and check that the external have the `(In English)` sting within them.
